shared/tracker/action.render.desktop.js

```diff
@@ -96,7 +97,7 @@ const styleContainer = {
 const styleActionButton = {
   width: 102,
   minWidth: 102,
-  marginRight: 0,
+  marginRight: globalMargins.xtiny,
```

Comment:
this has a side effect of having a margin when there's 1 button. this is why we have those visdiffs with one button when nothing should change